### PR TITLE
Cleanup most GPU buyer's guide pages

### PR DESCRIPTION
**Pull request by alyxferrari**

### CODE REVIEW

**Comment by alyxxxx** [comment]
Noticed some more things to change. Please don't merge this quite yet. Thanks.

**Comment by alyxxxx** [comment]
The Olarila link removed in the previous commit was dead.

File: modern-gpus/nvidia-gpu.md
Comment (1Revenger1) [inline modern-gpus/nvidia-gpu.md:187]:
```suggestion
* [Nvidia's Web Drivers](https://images.nvidia.com/mac/pkg/387/WebDriver-387.10.10.10.40.140.pkg)
```

File: modern-gpus/intel-gpu.md
Comment (1Revenger1) [inline modern-gpus/intel-gpu.md:19]: Note not needed anymore

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]
Thanks, this is much needed 👍

**Comment by alyxxxx** [comment]
> Thanks, this is much needed 👍
Glad to help out :)

Anyway, I'll get to work on the changes proposed in your review. Should be done by tonight or tomorrow.

File: README.md
Comment (alyxxxx) [reply README.md:13]: A pull request changing every "Nvidia" mention to "NVIDIA" in the NVIDIA buyer's guide was merged recently. I think that this along with the fact that NVIDIA is *technically* correct means that we should change it to NVIDIA when possible. Any thoughts on this?

File: modern-gpus/amd-gpu.md
Comment (alyxxxx) [reply modern-gpus/amd-gpu.md:64]: I've been very active in the AMD OS X server for almost a year and have never seen anyone complain about dual monitors not working on Navi, and it seems like if it weren't fixed, that would be a pretty common setup and I'd hear about it. Plus most Navi cards work without WEG anyway. That's why I removed it, I can put it back if you'd like.

**Comment by alyxxxx** [comment]
e6cdc0d

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md]: Resolved with 2362449 and e6cdc0d

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md:19]: > Note not needed anymore

Confused about what you'd like me to do :P

File: README.md
Comment (1Revenger1) [reply README.md:13]: I think their both fine. Wikipedia uses Nvidia instead of NVIDIA.
My thought is to generally go with Nvidia as it's less work :P

File: legacy-gpus/legacy-nvidia.md
Comment (alyxxxx) [reply legacy-gpus/legacy-nvidia.md]: Resolved with 8347188

File: modern-gpus/intel-gpu.md
Comment (1Revenger1) [reply modern-gpus/intel-gpu.md:19]:
```
* Note: As of macOS 11.3, the HD 4000 drivers are still present in Big Sur. Plase keep in mind....
```
They are in all versions of Big Sur, and were dropped in Monterey.
Probably need to update the Highest Support OS to Big Sur as well.

They weren't officially supported by Apple but continued to work 100% fine assuming you used a newer SMBIOS. Please don't use MacPro6,1 for a computer using the iGPU.

File: README.md
Comment (alyxxxx) [reply README.md:13]: image
They use NVIDIA on their site.
> I don't really care that much as long as it's consistent

The buyer's guide already uses it so I think I'm gonna go with keeping it here. Is that okay with you?

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md:19]: Gotcha. Forgot about Mac Pros not using iGPUs, I'll update that.

File: legacy-gpus/legacy-intel.md
Comment (alyxxxx) [reply legacy-gpus/legacy-intel.md]: Resolved with 8203db3

File: legacy-gpus/legacy-intel.md
Comment (alyxxxx) [reply legacy-gpus/legacy-intel.md]: Resolved with 8203db3

File: README.md
Comment (alyxxxx) [reply README.md]: Resolved with 446c8a1

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md]: Resolved with 3acbabc

File: README.md
Comment (alyxxxx) [reply README.md]: 32a9c57

**Comment by alyxxxx** [comment]
Looks like the only thing left to do is decide on whether to keep the Navi dual monitor WEG issue thread and waiting on Dortania to get back to you on the opinion piece in the introduction.

File: legacy-gpus/legacy-intel.md
Comment (1Revenger1) [inline legacy-gpus/legacy-intel.md:7]:
```suggestion
Unfortunately, Mojave dropped support for these iGPUs, but luckily we can actually get these iGPUs working by using old kexts (though no Metal support so things are a bit iffy). The [OpenCore Legacy Patcher](https://github.com/dortania/OpenCore-Legacy-Patcher/releases) is the best way to add support for these GPUs. However, OCLP is geared towards real Macs and while it will work on Hackintoshes, there is no official support. Intial support was introduced with macOS 10.7 and is not supported by the [Intel framebuffer patching guide](https://www.insanelymac.com/forum/topic/334899-intel-framebuffer-patching-using-whatevergreen/?tab=comments#comment-2626271)
```

File: modern-gpus/nvidia-gpu.md
Comment (1Revenger1) [inline modern-gpus/nvidia-gpu.md:226]:
```suggestion
* [NVIDIA's Web Drivers](https://images.nvidia.com/mac/pkg/387/WebDriver-387.10.10.10.40.140.pkg)
```

File: modern-gpus/intel-gpu.md
Comment (1Revenger1) [inline modern-gpus/intel-gpu.md:17]:
```suggestion
The HD4000 is supported natively up to macOS Big Sur, with support dropped in macOS Monterey. Note that you must be using an Ivy Bridge-compatible SMBIOS. The HD 2500, on the other hand, only has partial support in Mojave for quick sync features. As the HD 2500 does not support hardware acceleration, you will need a compatible dGPU for display purposes.
```

"Ivy Bridge-compatible SMBIOS" is a bit of a misnomer. We have SMBIOS recommendations in the install guide for both Ivy bridge desktops and laptops which will work. They're Haswell though.

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

**Comment by 1Revenger1** [comment]
I probably wouldn't mention SMBIOS too much - that's mentioned in a lot of other places such as the Big Sur page and the install-guide.

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md]: Resolved with bd390f8 and 3472111

File: README.md
Comment (1Revenger1) [reply README.md:17]: I'll work on this after merge

**Comment by 1Revenger1** [comment]
OCLP team requested that we link this instead: https://github.com/dortania/OpenCore-Legacy-Patcher/issues/348

File: modern-gpus/intel-gpu.md
Comment (1Revenger1) [inline modern-gpus/intel-gpu.md:5]: HEVC is required for some stuff like sidecar, not too sure about DRM.

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

File: modern-gpus/intel-gpu.md
Comment (alyxxxx) [reply modern-gpus/intel-gpu.md:5]: Yeah I don't think HEVC is required for DRM either, GCN 1.0 doesn't support HEVC but people are still able to get DRM working. What would you like me to do here? I removed it because I have a Kaby Lake laptop on macOS 11 and FairPlay 1, 2, and 3 work. Haven't tested FairPlay 4.

**Review by 1Revenger1 — APPROVED** [review]

**Pull request merged**
